src/mots/bmo.py

```diff
@@ -61,7 +61,7 @@ def get_users_by_ids(self, ids: list[str]):
         result = response.json()
         return {u["id"]: u for u in result["users"]}
 
-    def get_match(self, match: str):
+    def get_matches(self, match: str) -> list[dict]:
```

Comment:
Guess this was unused before, heh.

Would be nice to have a test for it.

Reply:
Yes, it was left as a remnant from an early implementation. It makes more sense to search by IDs for various reasons.